### ISSUE TITLE
support translating tW

### ISSUE THREAD

**Issue by neutrinog** [unfoldingWord-dev/ts-desktop#240]
see https://github.com/unfoldingWord-dev/ts-android/issues/451

**Comment by neutrinog** [comment]
the ui side will be similar to notes and questions. Each word will be a source card.

**Comment by neutrinog** [comment]
@jag3773 @cdwhitfield73  we need to discuss the tW data. There is a lot of duplication between Bible words and OBS words. Also how do we want to display this in the ui. e.g. Words are not necessarily related to a particular project.

**Comment by jag3773** [comment]
Hey @neutrinog and @cdwhitfield73, I discussed this with @benjore and we decided that we will only allow translating tW for the Bible, not OBS.  The OBS words are a subset of the Bible words, so if we get a translation of the tW for the Bible then we'll also have everything we need for OBS.

We could also move to a unified JSON catalog of the words, using tags to identify what project the word relates to.  That may not be necessary, but it may make things easier.

In either case, the Bible tW is what we want to provide as a source for translation.
